subprojects/plugin-use/src/main/java/org/gradle/plugin/use/internal/DefaultPluginRequestApplicator.java

```diff
@@ -102,8 +102,8 @@ public Result transform(PluginRequest request) {
             List<ArtifactRepository> pluginArtifactRepositories = new ArrayList<ArtifactRepository>();
             pluginRepositoryRegistry.lock();
             for (PluginRepository pluginRepository : pluginRepositoryRegistry.getPluginRepositories()) {
-                if (pluginRepository instanceof BackedByArtifactRepository) {
-                    pluginArtifactRepositories.add(((BackedByArtifactRepository) pluginRepository).createArtifactRepository(repositories));
+                if(pluginRepository instanceof BackedByArtifactRepositories) {
+                    pluginArtifactRepositories.addAll(((BackedByArtifactRepositories) pluginRepository).createArtifactRepositories(repositories));
```

Comment:
I was referring to this code earlier. We should just append the repositories instead of making the code more complicated to prepend them. Users shouldn't mix `pluginRepositories` and `buildscript.repositories` anyway.